.editorconfig

```diff
@@ -0,0 +1,220 @@
+###############################
```

Comment:
@TiagoBrenck 
I've added a .editorConfig which uses the same conventions as the .NET SDK team (and the .NET Core team, BTW, with little tweaks). This will help the code style to be consistent accross samples :)